src/features/settings/slice.js

```diff
@@ -59,9 +68,10 @@ const transformItemToSetting = ({ itemId, item, ...rest }) => {
     key = MENSES_TAG_KEY.SLICE
     value = textToTagArray(value)
   } else if (itemId === CYCLE_LENGTH_KEY.DB) {
-    // Will for some early users be stored as a string
+    // Will for some early user have an invalid value
+    const { value: cycleLength } = cycleLengthSchema.validate(value)
```

Comment:
Must make sure it all works when cycleLength is invalid and turns into `undefined`.